src/activities/assignments/AssignmentEntity.js

```diff
@@ -965,6 +964,9 @@ export class AssignmentEntity extends Entity {
 		if (assignment.hasOwnProperty('allowTextSubmission')) {
 			diffs.push([this.allowTextSubmission(), assignment.allowTextSubmission]);
 		}
+		if (assignment.hasOwnProperty('allowableFileType')) {
+			diffs.push([this.allowableFileTypeValue(), assignment.allowableFileType]);
+		}
```

Comment:
Only compare allowableFileType between saved entity and state now if the state that is being compared contains the property since with [this change](https://github.com/BrightspaceHypermediaComponents/activities/pull/3933/files#r1205913773) we will not always provide the allowableFileType